index.js

```diff
@@ -106,8 +112,9 @@ class CacheMem {
   }
 
   fallbackTll(key) {
+    let ttl = Math.max((this.localTtlCache[key] || 0) - Date.now(), 0);
     return new Promise(resolve => {
-      resolve(this._defaultExpiration);
+      resolve(ttl);
```

Comment:
Can we make it work with 2.8? Or maybe in another PR?

```
Starting with Redis 2.8 the return value in case of error changed:
The command returns -2 if the key does not exist.
The command returns -1 if the key exists but has no associated expire.
See also the PTTL command that returns the same information with milliseconds resolution (Only available in Redis 2.6 or greater).
```